### PR TITLE
Sort Museums by Rating and Update on the Show Page

### PR DESCRIPTION
**Pull request by ChrisDonohue0628**

### CODE REVIEW

File: react/src/components/MuseumsContainer.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
```
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: needed for React

File: react/src/components/MuseumsContainer.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
```
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: needed for React

File: react/src/components/MuseumsContainer.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+import Museum from './Museum';
```
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: needed for React

File: react/src/components/MuseumsContainer.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+class MuseumsContainer extends Component {
+  constructor(props) {
```
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: needed for React

File: react/src/components/MuseumsContainer.js (hunk 3 above)
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: needed for React

File: react/src/components/MuseumsContainer.js (hunk 2 above)
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: needed for React

File: react/src/components/MuseumsContainer.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import Museum from './Museum';
```
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: needed for React

File: react/src/containers/MuseumsContainer.js [hunk 6]
```diff
@@ -10,13 +10,28 @@ class MuseumsContainer extends Component {
...
+      comparison = 1;
+    } else if (ratingA > ratingB) {
+      comparison = -1;
+    }
+    return comparison;
+  }
+
   getData () {
-    fetch('https://localhost:3000/api/v1/museums.json')
+    let thiserino = this
```
Comment (smkopp92) [inline react/src/containers/MuseumsContainer.js]: I know thiserino works, but I would use another variable name, like maybe MuseumContainer

**Pull request merged**